### ISSUE TITLE
Public 2000+ Records Notifications

### ISSUE THREAD

**Issue by emjohnst** [bcgov/BCHeritage#1124]
When the public exports more than 2000 records, the download notification is displayed to any user. Any user can also clear the notifications making the statement of when the download is available until redundant if any other public user accesses this. 

Options:
Either disable 2000 record export for public
Or
Explore options so downloads and notifications are only viewable to the specific public user that requested it

**Comment by bferguso** [comment]
@emjohnst - I spoke to the Arches Core team and they've never though to implement a session-based filter for the export results, so for public users the current functionality is as designed (even if the behaviour is not what we want). I think we should disable the 2000 for now and then look at implementing a session-based filter sometime after the 7.6.x release. Does that work?

**Comment by emjohnst** [comment]
@bferguso yes let's create a session based filter, in the meantime can we leave the 2000 records enabled?

**Comment by bferguso** [comment]
@emjohnst - Sure, lets just repurpose this issue for the session-based filter issue, and push it back. Nothing to do for this release then.
